red.py

```diff
@@ -154,6 +154,9 @@ def user_allowed(self, message):
         else:
             return True
 
+    def prefix_manager(self, bot, msg):
```

Comment:
As discussed on the server, the callable needs to be exactly like that or it won't work